### PR TITLE
Mathbox

### PR DESCRIPTION
**Pull request by benjub**
Mathbox only. See detailed commit message.

### CODE REVIEW

**Comment by benjub** [comment]
What's up with the changes in the other files ? I have not touched them in my commit... Trying to restore them...

**edit:** It turns out it was intermediate files mistakenly added. I just removed them.

File: set.mm [hunk 1]
```diff
@@ -531652,17 +531554,17 @@ written as (is equal to) a class abstraction does not require these
...
+       proof of either); plus, it is of the same form as ~ cvjust and such a
+       basic statement deserves to be present in both forms.  Note that
+       ~ bj-termab shortens the proof of ~ abid2 , and shortens five proofs by
+       a total of 72 bytes.  Move it to Main as "abid1" proved from ~ abbi2i ?
+       Note also that this is the form in Quine, more than ~ abid2 .
+       (Contributed by BJ, 21-Oct-2019.)
        (Proof modification is discouraged.) $)
     bj-termab $p |- A = { x | x e. A } $=
-      ( cv wcel biid bj-abbi2i ) ACBDZABGEF $.
+      ( cv wcel biid abbi2i ) ACBDZABGEF $.
```
Comment (icecream17) [inline set.mm:531567]: incidentally this is identical to abid1

File: set.mm [hunk 2]
```diff
@@ -533996,45 +533958,42 @@ automatically a set (see ~ bj-mooreset ).  (Contributed by BJ,
...
+      VDVCVCVBUBUCVCVDCUDTUETUAUFUNUGAVAUHZVEOUSVKURIQZAVEURIUOAVBCNVLVDVBQVEDV
+      BURUIVBCVDUJUKULUMUPUQ $.
   $}
 
   ${
     $d x A $.
-    $( The discrete Moore collection on a set.  (Contributed by BJ,
-       9-Dec-2021.) $)
+    $( The powerclass ` ~P A ` is a Moore collection if and only if ` A ` is a
+       set.  It is then call the discrete Moore collection.  (Contributed by
```
Comment (icecream17) [inline set.mm:533991]:
```suggestion
       set.  It is then called the discrete Moore collection.  (Contributed by
```

**Review by icecream17 — COMMENTED** [review]
I'm only halfway through the review right now, will finish tomorrow. (Edit: oops...)

File: set.mm [hunk 3]
```diff
@@ -534383,184 +534530,351 @@ Identity relation (complements)
...
+
+  ${
+    $d x y A $.  $d x y B $.
+    $( Sufficient condition for the restricted identity relation to agree with
+       equality.  Note that the instance of ~ bj-ideqg with ` _V ` substituted
+       for ` V ` is a direct consequence of ~ bj-idreseq .  This is a
+       strengthening of ~ resieq which should be proved from it (note that
+       currently, ~ resieq relies on ~ ideq ).  Note that the intersection in
+       the antecedent is not very meaningful, but is a device to prove versions
+       with either class assumed to be a set.  I could be enough to prove the
```
Comment (avekens) [inline set.mm:534667]: " I could" -> "It could"?

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm]: Thanks: this was actually moved to Main as ~ abid1 as proposed, but I forgot to remove it here. Fixed.

File: set.mm (hunk 2 above)
Comment (benjub) [reply set.mm:533991]: Thanks. Fixed.

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:534667]: Thanks. Fixed.

**Review by avekens — APPROVED** [review]

**Pull request merged**